parts/k8s/kubernetesparams.t

```diff
@@ -240,7 +240,7 @@
       }
     },
     "mobyVersion": {
-      "defaultValue": "3.0.8",
+      "defaultValue": "3.0.9+azure-38",
```

Comment:
why the `+azure-38` tag?

Reply:
`+azure` is an extra hint to users that it's not some really old version of docker.
The `-38` is the version of the package, essentially. So we can change something package related only and not bump engine versions.

I was hoping `apt-get install moby-engine=3.0.9` would work with this scheme it it apparently wants the full version only.
This is likely to get even more complicated as we move away from static compilation and have packages targeted for specific distro versions.

Reply:
What I'm going to recommend is that we create a metapackage like `aks-engine-moby` which can hold all the specific dependencies for a given version.
So we'd a package with the desired simple version (e.g. 3.0.9), inside the package it's just a list of dependencies and their versions. This also fixes the issue of having to keep CLI and engine versions in lock step.

Reply:
WDYT?